### DISCUSSION TITLE
Overriding font size

### DISCUSSION THREAD

**Discussion by LukazTrisauskas** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3603] [Q&A]
I'm new to WPF and just trying out this toolkit. I've tried to follow the guide on how to override default styles, but can't figure out how to override the font size.

Adding the follow code to the windows properties doesn't have an affect on the font size either.

`TextElement.FontSize="11"` or `FontSize="11"`. I can easily change the `FontFamily` through Window properties, but not the size, which is strange.

Any help would be appreciated.

**Comment by MichelMichels** [comment]
Hi! Welcome fellow WPF enthusiast! I'll make a guess that the font size you want changed is for `TextBlock` controls? Each control has its own style defined (almost each). For text block, the font size is set inside the style. If you want to change this, you must alter that style, controls don't automatically inherit styles from their parent. 

If you want to change fontsizes for all text block controls in your window, you can write following xaml:
```xml
<Window ...>
<Window.Resources>
<Style TargetType="TextBlock">
<Setter Property="FontSize" Value="24" />
</Style>
</Window.Resources>

// Window content
```

I wrote this on my phone, so syntax errors are possible.
